UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -1062,9 +1076,8 @@ <h5>Instructions</h5>
 							<ol class="condition">
 								<li><b>LET</b> <var>adaptation_string</var> be the result of:
 									<ul class="condition">
-										<li>calling join on <var>adaptation</var> with the separator <code>", "</code></li>
-										<li>making uppercase the first character</li>
-										<li>replacing the last occurrence of <code>", "</code> with <code id="join-array-and">", and "</code>.</li>
+										<li>calling <a href="#join-array-to-comma-list">join array to comma list</a> given <var>adaptation</var></li>
+										<li>uppercasing the first character.</li>
```

Comment:
Sam here about uppercase

Reply:
Like here it says it is going to capitalize the first letter, but if it doesn't we are confusing people so we should just remove that line.